### ISSUE TITLE
Fills are not showing previews in params nor simulator

### ISSUE THREAD

**Issue by LinaX23** [inkstitch/inkstitch#1630]
working on embroidering a picture using satin and fill stitches. satin stitches are working fine and show up everywhere. my previous fill stitches are not showing previews in params nor the simulator. Params pops up but only the first window opens where you can change the settings, and simulator just keeps loading. I tried adding a new fill object and it worked but why aren't my previous ones. I had this problem and just started over in a new document it worked for a bit but now I'm having problems again.
![The flower](https://user-images.githubusercontent.com/103866319/163696786-dba53450-30b9-43fd-a151-5d5091801df9.svg)

**Comment by tyrosinase** [comment]
Try `Extensions > Ink/Stitch > Troubleshoot > Troubleshoot Objects` - you've got some border issues that need cleaned up. (Sometimes that error just means there are too many nooks and crannies where stitches don't fit rather than a border that fully crosses over itself.)

**Comment by kaalleen** [comment]
Open the objects panel.

All objects in the layer "leaves" and in the layer "flower" have both a stroke and a fill, while they should only have a fill. Select all these objects and click on the most left color swatch with the red cross while holding the shift key down.

![image2808](https://user-images.githubusercontent.com/36401965/163783461-902e65c4-cf4a-4e9a-8cdb-cf4b2d736334.png)
 
I learned today, that GitHub now allows to upload svg files directly. This is so good for us. Maybe now we finally will receive more svg files for inspection :) Thank you for posting your issue along with an svg file, this makes it so much easier for us to answer questions!!!

**Comment by LinaX23** [comment]
I did trouble shoot and it says all shapes valid. This is weird because when I made all the shapes to begin with and I checked them they worked. Also I just redid the leaves and put them together and params ran properly then I went to change the color and tried to view in the simulator and the simulator did not load it. So I closed it out ran the params again and now its no working. I have created many things using inkstitch so I'm familiar with it but this has never happened before.
